.github/workflows/release.yml

```diff
@@ -1,5 +1,8 @@
 name: release
 on:
+  workflow_run:
+    workflows: publish-dist
+    types: [completed]
   push:
     tags:
```

Comment:
This doesn't quite work -- the workflow will run *both* on pushing the tag, *and* for *every* PR opening and landing.

Rather, keep the `push: tags` trigger and teach the download-dist call in `Makefile` something like `${DOWNLOAD_DIST_OPTIONS:-}` and in this workflow set `DOWNLOAD_DIST_OPTIONS=--wait`.

Reply:
I tested it locally, outside of the release workflow and it works like a charm

I am not sure how I should actually test it by triggering the real release-workflow, I don't want to publish something accidentally from a run from my fork.